.cico/cico_openshift_e2e.sh

```diff
@@ -9,41 +9,39 @@
 #
 set -ex
 
-# Setup to find necessary data from cluster setup
-export CI="openshift" # ENV used by PROW ci to get the oc client
+# ENV used by PROW ci
+export CI="openshift" 
 export ARTIFACTS_DIR="/tmp/artifacts"
-export CUSTOM_HOMEDIR=$ARTIFACTS_DIR
 
-# Overridable information
-export DEFAULT_INSTALLER_ASSETS_DIR=${DEFAULT_INSTALLER_ASSETS_DIR:-$(pwd)"/.e2e"}
-export KUBEADMIN_USER=${KUBEADMIN_USER:-"kubeadmin"}
-export KUBEADMIN_PASSWORD_FILE=${KUBEADMIN_PASSWORD_FILE:-"${DEFAULT_INSTALLER_ASSETS_DIR}/auth/kubeadmin-password"}
+# Pod created by openshift ci don't have user. Using this envs should avoid errors with git user.
+export GIT_COMMITTER_NAME="CI BOT"
+export GIT_COMMITTER_EMAIL="ci_bot@notused.com"
 
-# Exported to current env
-export KUBECONFIG=${KUBECONFIG:-"${DEFAULT_INSTALLER_ASSETS_DIR}/auth/kubeconfig"}
+# Check if operator-sdk is installed and if not install operator sdk in $GOPATH/bin dir
+if ! hash operator-sdk 2>/dev/null; then
+    mkdir -p $GOPATH/bin
+    export PATH="$PATH:$(pwd):$GOPATH/bin"
+    RELEASE_VERSION=v0.17.0
```

Comment:
Let's rename it to OPERATOR_SDK_VERSION?

Reply:
Sure.